CHANGES.md

```diff
@@ -4,6 +4,12 @@ Change Log
 ### Next Release
 
 * cacheOptimization no longer crashes on primitives without indices. [#154](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/154)
+* Public API is exposed via index.js [#153](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/153)
+  * Documentation has been added for all exposed functions.
+  * OptimizationStats is removed from removeUnusedStages.
```

Comment:
Throughout CHANGES.md use, for example, `OptimizationStats` not just OptimizationStats for classes/functions/etc.